sampleModel/src/samplemodel/SampleModelOperationRegistry.java

```diff
@@ -45,8 +45,8 @@ OperationLibrary getOperationLibraryRoot()
       operationLibraryRoot = new OperationLibrary();
 
       // read all configuration elements and split them into libraries/operations
-      IConfigurationElement[] configurationElements = Platform
```

Comment:
A few changes in this file are due to reformatting - due to the preferred line width.  Have you switched PC? 

Ideally, our commits should just include "changes of substance" - it makes them much easier to review :-)